src/Components/CustomSeparator.jsx

```diff
@@ -20,6 +21,12 @@ navMap.set("/license/generate", {
   title: "Generate Product Key"
 });
 
+navMap.set(LICENSE_SEARCH_PAGE_ROUTE, {
```

Comment:
Can the nav map be moved to routes. I think this is used in several components

Reply:
Will do in separate PR